.neutrinorc.js

```diff
@@ -1,56 +1,106 @@
 const fs = require('fs-extra');
 const { join } = require('path');
 
-require('babel-register')({
-  plugins: [
-    [require.resolve('babel-plugin-transform-es2015-modules-commonjs'), {
-      useBuiltIns: true
-    }],
-    require.resolve('babel-plugin-transform-object-rest-spread'),
-  ],
+require('@babel/register')({
```

Comment:
You can remove the use of register and preset-env entirely from neutrinorc if you switch `theme.js` to be CommonJS instead of ES imports. The `theme.js` isn't used by the components, so will be safe to do from Node.js.